src/mbodied_agents/agents/motor/motor_agent.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC, abstractmethod
+from typing import List
+
+from mbodied_agents.types.controls import Motion
+
+
+class MotorAgent(ABC):
```

Comment:
Let's call it MotionAgent. Also rename the directory.

Reply:
Initially it was MotionAgent. @sebbyjp suggested MotorAgent